list  server url for proxy for different sdks (#538)
Amplitude-Developer-Docs/amplitude-dev-center: docs/analytics/domain-proxy.md

### Configure the SDKs to work with alternate endpoints

> After the proxy is working correctly, configure your SDK. Amplitude’s SDKs are open source code that you can change and have built-in options to send events to your defined server endpoint

Reviewer: ```suggestion
After the proxy is working correctly, configure your SDK. Amplitude’s SDKs are open source code that you can change and have built-in options to send events to your defined server endpoint.
```

> After the proxy is working correctly, configure the SDK. Amplitude’s SDKs are open source code that you can change. The SDKs already have built-in options to send events to your defined server endpoint.

Reviewer: @yuhao900914 sorry for the out of scope suggestion here, but this has always read weird to me. 

Can we rephrase this to something like: 

```suggestion
After the proxy is working correctly, configure your SDK. Amplitude’s SDKs are open source code that you can change and have built-in options to send events to your defined server endpoint.
```

> !!!note

Reviewer: ```suggestion

```

>     Different SDKs point to different API endpoint with its custom payload. The following list are the endpoints for different SDKs.

Reviewer: ```suggestion
The SDKs point to special endpoints for their custom payloads. Find the endpoint for your SDK in this table: 
```

>  <div class="big-column">SDK</div> | Endpoint | Set server url |

Reviewer: ```suggestion
 |<div class="big-column">SDK</div> | Endpoint | Set server url |
```

fixes broken table

| [Amplitude-Android](../data/sdks/android/)| `https://api2.amplitude.com/` | Use the `setServerUrl` function to configure the server URL. |
| [Amplitude-Java](../data/sdks/java/) | `https://api2.amplitude.com/2/httpapi` | Use the `setServerUrl` function to configure the server URL. |
| [Amplitude-iOS](../data/sdks/ios/) | `https://api2.amplitude.com/` | Use the `setServerUrl` function to configure the server URL. |
| [Amplitude-TypeScript](../data/sdks/typescript-browser/) | `https://api2.amplitude.com/2/httpapi` | Set the `serverUrl` option when initializing the SDK. |
| [Amplitude-Kotlin](../data/sdks/android-kotlin/) | `https://api2.amplitude.com/2/httpapi` | Set the `serverUrl` option when initializing the SDK. |
| [Amplitude-Swift](../data/sdks/ios-swift/) | `https://api2.amplitude.com/2/httpapi` | Set the `serverUrl` option when initializing the SDK. |
| [Amplitude-Python](../data/sdks/python/) | `https://api2.amplitude.com/2/httpapi` | Set the `server_url` option when initializing the SDK.|
| [Amplitude-Go](../data/sdks/go/) | `https://api2.amplitude.com/2/httpapi` |  Set the `serverUrl` option when initializing the SDK. |

> Now you can send events through your proxy and see them logged in Amplitude.

Reviewer: ```suggestion
After you configure the SDK, you can send events through your proxy and see them logged in Amplitude.
```
